bin/check-nvmrc.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+
+'use strict'
+
+var fs = require('fs')
+var path = require('path')
+
+fs.readFile(path.join(__dirname, '../.nvmrc'), 'utf8', function (error, data) {
+  if (error) throw error
+  var expectedVersion = data.trim()
+  var currentVersion = process.version.replace('v', '')
+
+  var versionMatchesExactly = expectedVersion === currentVersion
+  var versionMatchesMajor = expectedVersion.substring(1, 0) === currentVersion.substring(1, 0)
```

Comment:
I think this will break as soon as we get to node 10?

Reply:
`.split('.')[0]` might work better

Reply:
Fair point, will update.